lolcommits.gemspec

```diff
@@ -42,6 +42,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('launchy', '~> 2.4.3')
   s.add_runtime_dependency('methadone', '~> 1.8.0')
   s.add_runtime_dependency('open4', '~> 1.3.4')
+  s.add_runtime_dependency('mercurial-ruby', '~> 0')
```

Comment:
We'll probably want to pin to a more constrained version number.  (I'm also slightly concerned that there hasn't been an update to this dep since 2013, but not sure what alternative paths might exist.)

(Thankfully this gem doesn't introduce any additional new sub-deps, as it only has open4 which we already require.)